e2e/test/framework/pod.go

```diff
@@ -108,6 +109,16 @@ func (f *Invocation) WaitForReady(meta metav1.ObjectMeta) error {
 	})
 }
 
+func (f *Invocation) WaitForDelete(meta metav1.ObjectMeta) error {
+	return wait.PollImmediate(f.RetryInterval, f.Timeout, func() (bool, error) {
+		_, err := f.kubeClient.CoreV1().Pods(meta.Namespace).Get(meta.Name, metav1.GetOptions{})
+		if apierrors.IsNotFound(err) {
+			return false, nil
+		}
+		return false, nil
```

Comment:
`, err`?

Reply:
ya that would work well if im catching the is not found, good idea